File: src/ruby etc/fsociety.rb
require 'time'
require 'digest'

puts <<~BANNER
    .o88o.                               o8o                .
    888 `"                               `"'              .o8
   o888oo   .oooo.o  .ooooo.   .ooooo.  oooo   .ooooo.  .o888oo oooo    ooo
    888    d88(  "8 d88' `88b d88' `"Y8 `888  d88' `88b   888    `88.  .8'
    888    `"Y88b.  888   888 888        888  888ooo888   888     `88..8'
    888    o.  )88b 888   888 888   .o8  888  888    .o   888 .    `888'
   o888o   8""888P' `Y8bod8P' `Y8bod8P' o888o `Y8bod8P'   "888"      d8'
                                                                .o...P'
                                                                `XER0'
BANNER

loop do
  puts "\n--------------------"
  puts "help - show commands"
  puts "--------------------"

  print "┌──(fsociety㉿kali)\n└─$ "
  choice = gets.chomp.downcase.strip

  if choice == "help"
    puts "\n------------------------"
    puts "  fastfetch - show specs  "
    puts "   github - show source   "
    puts "   infos - show updates   "
    puts "   exit - exit program    "
    puts "   ddos - ddos attack     "
    puts "   hack - payload hack    "
    puts " bruteforce - brute force "
    puts "--------------------------"

  elsif choice == "fastfetch"
    puts <<~SPECS
        ..............                                      fsociety@kali
             ..,;:ccc,.                              ---------------------------
           ......''';lxO.                            OS: Kali GNU/Linux Rolling x86_64
 .....''''..........,:ld;                            Host: 20W1S7PR18 (ThinkPad T14 Gen 2i)
            .';;;:::;,,.x,                           Kernel: Linux 6.18.12+kali-amd64
       ..'''.            0Xxoc:,.  ...               Uptime: 2 hours, 11 mins
   ....                ,ONkc;,;cokOdc',.             Packages: 3568 (dpkg)
  .                   OMo           ':ddo.           Shell: zsh 5.9
                     dMc               :OO;          Display (AUO323D): 1920x1080 in 14", 60 Hz [Built-in]
                     0M.                 .:o.        DE: Xfce4 4.20
                     ;Wd                             WM: Xfwm4 (X11)
                      ;XO,                           WM Theme: Kali-Dark
                        ,d0Odlc;,..                  Theme: Fusion [Qt], Kali-Dark [GTK2/3/4]
                            ..',;:cdOOd::,.          Icons: Flat-Remix-Blue-Dark [Qt], Flat-Remix-Blue-Dark [GTK2/3/4]
                                     .:d;.':;.       Font: Cantarell (11pt) [GTK2/3/4]
                                        'd,  .'      Cursor: Adwaita (24px)
                                          ;l   ..    Terminal: qterminal 2.3.0
                                           .o        Terminal Font: FiraCode (10pt)
                                             c       CPU: 11th Gen Intel(R) Core(TM) i5-1145G7 (8) @ 4.40 GHz
                                             .'      GPU: Intel Iris Xe Graphics @ 1.30 GHz [Integrated]
                                              .      Memory: 2.06 GiB / 15.33 GiB (13%)
                                                     Swap: 0 B / 12.30 GiB (0%)
                                                     Disk (/): 93.97 GiB / 220.63 GiB (43%) - ext4
                                                     Local IP (wlan0): 192.168.178.35/24
                                                     Battery (5B10W51826): 100% [Discharging]
                                                     Locale: nl_NL.UTF-8
    SPECS

  elsif choice == "github"
    puts "\nhttps://github.com/AeroXPtech"

  elsif choice == "infos"
    puts "added payload hack and ransomware simulator"

  elsif choice == "exit"
    puts "\nGood Bye!"
    exit(0)

  elsif choice == "ddos"
    print "choose Website (ex. usa.gov): "
    site = gets.chomp

    puts "\nGood, which botnet do you want to use?"
    puts "Mira (1.7M devices)"
    puts "Ikai (764K devices)"
    puts "Yuro (567K devices)"
    puts "Liro (387K devices)"
    print "→ "
    botnet = gets.chomp

    puts "\nGreat! wait while #{botnet} is doing their work! (Est. 30sec-2min)"
    sleep(2)

    15.times do
      puts "DDoSing #{site}. 200 OK"
      sleep(2)
    end

    puts "\nDDoS finished! 503 Service Unavailable"
    sleep(2)

  elsif choice == "hack"
    print "\nchoose IP (ex. 129.983.38:723 ): "
    ip = gets.chomp

    puts "\nGood, what Payload do you want to use?"
    puts "1 - Luma3DS"
    puts "2 - Ja1lh4ck1ng"
    puts "3 - dualpayload"
    puts "4 - payload.cia"
    puts "5 - Choose own exploit"
    print ">>> "
    payload = gets.chomp

    puts "\nGreat! wait while #{payload} is Injecting The Hack (Est ca 1 min)"
    sleep(5)
    puts "Starting Hack"
    sleep(2)
    puts "loading Payload"
    sleep(2)
    puts "loading Kernel"
    sleep(5)
    puts "patching kernel"
    sleep(2)
    puts "Injecting Payload"
    sleep(5)
    puts "rebooting..."
    sleep(10)
    puts "finishing Hack"
    sleep(6)
    puts "Successfully Hacked #{ip}!"

  elsif choice == "bruteforce"
    common_passwords = [
      "123456", "password", "123456789", "qwerty", "12345678", "12345", "1234567", "password123",
      "admin", "root", "letmein", "welcome", "monkey", "dragon", "master", "shadow", "fsociety",
      "elliot", "alderson", "kali", "hacker", "123123", "qwerty123", "abc123", "password1",
      "111111", "1234", "superman", "batman", "killer", "matrix", "secret", "football"
    ]

    target_hash = "6f52550186105d6a2f7c006769062947"

    puts "\e[1;33m[!] INITIALIZING BRUTE FORCE ATTACK ON HASH: #{target_hash}\e[0m"
    sleep(1)

    found = false

    common_passwords.each do |password|
      print "\r\e[1;34m[*] TRYING:\e[0m #{password.ljust(15)}"
      $stdout.flush

      sleep(rand(0.3..0.8))

      guess_hash = Digest::MD5.hexdigest(password)

      if guess_hash == target_hash
        puts "\n\n\e[1;32m[+] MATCH FOUND!\e[0m"
        puts "\e[1;32m[+] PASSWORD:\e[0m #{password}"
        found = true
        break
      end
    end

    puts "\n\e[1;31m[-] ATTACK FAILED: Password not in dictionary.\e[0m" unless found

  else
    puts "\nIllegal Command"
  end
end

File: src/Python/fsociety.py
import time
import os

print("""
    .o88o.                               o8o                .
    888 `"                               `"'              .o8
   o888oo   .oooo.o  .ooooo.   .ooooo.  oooo   .ooooo.  .o888oo oooo    ooo
    888    d88(  "8 d88' `88b d88' `"Y8 `888  d88' `88b   888    `88.  .8'
    888    `"Y88b.  888   888 888        888  888ooo888   888     `88..8'
    888    o.  )88b 888   888 888   .o8  888  888    .o   888 .    `888'
   o888o   8""888P' `Y8bod8P' `Y8bod8P' o888o `Y8bod8P'   "888"      d8'
                                                                .o...P'
                                                                `XER0'
     """)

while True:
    print("\n--------------------")
    print("help - show commands")
    print("--------------------")

    choice = input("""┌──(fsociety㉿kali)
└─$ """).lower()

    if choice == "help":
        print("\n------------------------")
        print("fastfetch - show specs    ")
        print("github - show source      ")
        print("infos - show updates      ")
        print("exit - exit programm      ")
        print("ddos - ddos attack        ")
        print("hack - payload hack       ")
        print("ransom - simulator        ")
        print("bruteforce - brute force  ")
        print("firewall - enable or dis..")
        print("--------------------------")

    elif choice == "fastfetch":
        print("""
        ..............                                      fsociety@kali
             ..,;:ccc,.                              ---------------------------
           ......''';lxO.                            OS: Kali GNU/Linux Rolling x86_64
 .....''''..........,:ld;                            Host: 20W1S7PR18 (ThinkPad T14 Gen 2i)
            .';;;:::;,,.x,                           Kernel: Linux 6.18.12+kali-amd64
       ..'''.            0Xxoc:,.  ...               Uptime: 2 hours, 11 mins
   ....                ,ONkc;,;cokOdc',.             Packages: 3568 (dpkg)
  .                   OMo           ':ddo.           Shell: zsh 5.9
                     dMc               :OO;          Display (AUO323D): 1920x1080 in 14", 60 Hz [Built-in]
                     0M.                 .:o.        DE: Xfce4 4.20
                     ;Wd                             WM: Xfwm4 (X11)
                      ;XO,                           WM Theme: Kali-Dark
                        ,d0Odlc;,..                  Theme: Fusion [Qt], Kali-Dark [GTK2/3/4]
                            ..',;:cdOOd::,.          Icons: Flat-Remix-Blue-Dark [Qt], Flat-Remix-Blue-Dark [GTK2/3/4]
                                     .:d;.':;.       Font: Cantarell (11pt) [GTK2/3/4]
                                        'd,  .'      Cursor: Adwaita (24px)
                                          ;l   ..    Terminal: qterminal 2.3.0
                                           .o        Terminal Font: FiraCode (10pt)
                                             c       CPU: 11th Gen Intel(R) Core(TM) i5-1145G7 (8) @ 4.40 GHz
                                             .'      GPU: Intel Iris Xe Graphics @ 1.30 GHz [Integrated]
                                              .      Memory: 2.06 GiB / 15.33 GiB (13%)
                                                     Swap: 0 B / 12.30 GiB (0%)
                                                     Disk (/): 93.97 GiB / 220.63 GiB (43%) - ext4
                                                     Local IP (wlan0): 192.168.178.35/24
                                                     Battery (5B10W51826): 100% [Discharging]
                                                     Locale: nl_NL.UTF-8
                                                     0x10 0x20 0x30 0x40 0x50 0x60 0x67

                                                                              
                                                                             """)

    elif choice == "github":
        print("\nhttps://github.com/AeroXPtech")

    elif choice == "infos":
        print("added payload hack and ransomware simulator")

    elif choice == "exit":
        print("\nGood Bye!")
        exit()

    elif choice == "rootkit":
        import sys
        import random
        import hashlib

        def simulate_brute_force(target_hack):
            common_passwords = [
            "123456", "password", "123456789", "qwerty", "12345678", "12345", "1234567", "password123", "1234567890", "1234",
            "admin", "p@ssword", "root", "111111", "monkey", "sunshine", "iloveyou", "letmein", "secret", "football",
            "dragon", "master", "shadow", "hunter", "superman", "batman", "killer", "matrix", "hacker", "fsociety",
            "welcome", "login", "guest", "access", "system", "security", "server", "network", "firewall", "database",
            "000000", "123123", "654321", "987654321", "password1", "pass123", "abc123", "qazwsx", "wsxedc", "edcrfv",
            "chelsea", "arsenal", "liverpool", "united", "barcelona", "madrid", "soccer", "baseball", "basketball", "hockey",
            "charles", "george", "thomas", "william", "alexander", "nicholas", "matthew", "andrew", "daniel", "robert",
            "jennifer", "jessica", "michelle", "amanda", "ashley", "sarah", "stephanie", "heather", "nicole", "elizabeth",
            "princess", "angel", "sweetie", "honey", "cookie", "maggie", "charlie", "bailey", "lucky", "buster",
            "laptop", "computer", "keyboard", "monitor", "iphone", "android", "windows", "linux", "ubuntu", "kali", "elliot",
            "alderson", "elliotalderson", "verstappen", "hamilton", "starwars", "avengers", "pokemon", "superstar", "freedom", "awesome",
            "keyboard", "asdfghjk", "mnbvcxz", "1q2w3e4r", "password!", "P@ssw0rd1", "user1234", "test1234", "admin123", "manager",
            "spring2024", "summer24", "winter25", "january", "monday", "coffee", "whiskey", "tequila", "beer123", "smoke",
            "quantum", "entropy", "algorithm", "localhost", "127.0.0.1", "sysadmin", "database1", "oracle", "cisco123", "security!",
            "beowulf", "ragnarok", "anubis", "zeus123", "olympus", "viking", "sparta", "gladiator", "phoenix", "valhalla",
            "carpediem", "mementomori", "cogitoergo", "fortuna", "virtue", "justice", "liberty", "victory", "champion", "winner"
        ]

        print("Where Do you want to Eject Rootkit?")
        time.sleep(3)
        print("root")
        time.sleep(3)
        print("home")
        time.sleep(3)
        print("everywhere")
        time.sleep(3)
        print("own location")
        time.sleep(4)
        print("\nKeep it Empty For Root Location")
        local = input("/")
        time.sleep(3)
        print("Great! Wait while The Rootkit is Ejecting in /" + local + " ")
        time.sleep(2)
        print("Password Hack Required continue?")
        option = input()
        print(""+option+" was Picked")
        time.sleep(4)
        print("\n\033[1;31m[/]COULDN'T KILL PROSESS... CONTINUING\033[0m")
        time.sleep(3)
        print("May Take A While")
        time.sleep(45)
        print("Password Found!")
        print("Password: KIRBY1213FSBLABLABLA.")
        print("Attempt 24360 From 2045364362")
        time.sleep(3)
        print("Loading Payload via Exploit")
        time.sleep(5)
        print("SUCCESSFULLY ROOTED " + local + "")
        print("Target May Get a Email!!!")
        time.sleep(4)

    elif choice == "ddos":
       print("choose Website (ex. usa.gov)")
       site = input()
       print("Good, which botnet do you want to use?")
       print("Mira (1.7M devices)")
       print("Ikai (764K devices)")
       print("Yuro (567K devices)")
       print("Liro (387K devices)")
       botnet = input()
       print("Great! wait while " + botnet + " is doing their work! (Est. 30sec-2min)")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(2)
       print("DDoSing " + site + ". 200 OK")
       time.sleep(5)
       print("DDoS finished! 503 Service Unavailable")
       time.sleep(2)

    elif choice == "hack":
        print("\nchoose IP (ex. 129.983.38:723 )")
        ip = input(">>> ")
        print("\nGood, what Payload do you want to use")
        print("1 - Luma3DS")
        print("2 - Ja1lh4ck1ng")
        print("3 - dualpayload")
        print("4 - payload.cia")
        print("5 - Choose own exploit")
        payload = input(">>> ")
        print("\nGreat! wait while " + payload + " is Injecting The Hack (Est ca 1 min)")
        time.sleep(5)
        print("Starting Hack")
        time.sleep(2)
        print("loading Payload")
        time.sleep(2)
        print("loading Kernel")
        time.sleep(5)
        print("patching kernel")
        time.sleep(2)
        print("Injecting Payload")
        time.sleep(5)
        print("rebooting...")
        time.sleep(10)
        print("finishing Hack")
        time.sleep(6)
        print("Successfully Hacked "+ ip +"!")
        
    elif choice == "ransom":
        import tkinter as tk

        class RansomSim:
            def __init__(self, root):
                self.root = root
                self.root.title("SECURITY ALERT")
                self.root.attributes("-fullscreen", True)  
                self.root.configure(bg="black")             
                self.label = tk.Label(root, text="YOUR FILES ARE ENCRYPTED!", 
                                      fg="red", bg="black", font=("Helvetica", 36, "bold"))
                self.label.pack(expand=True)
                self.info = tk.Label(root, text="PAY 0.032 BITCOIN TO THIS ADDRESS TO UNENCRYPT THEM: bc1p6aktp2ruufxuqgv9lfwwgajlckxg2srlle3fe70umrzyu23rhfjsx9jzgx", 
                                     fg="white", bg="red", font=("Helvetica", 18))
                self.info.pack(pady=20)
                self.time_left = 86400  
                self.timer_label = tk.Label(root, text="", fg="white", bg="red", 
                                            font=("Helvetica", 48, "bold"))
                self.timer_label.pack(expand=True)
                self.update_timer()

            def update_timer(self):
                hours, remainder = divmod(self.time_left, 3600)
                minutes, seconds = divmod(remainder, 60)
                time_str = f"{hours:02}:{minutes:02}:{seconds:02}"
                self.timer_label.config(text=time_str)
                if self.time_left > 0:
                    self.time_left -= 1
                    self.root.after(1000, self.update_timer)

        root = tk.Tk()
        app = RansomSim(root)
        root.mainloop()
        
    elif choice == "bruteforce":
        import sys
        import random
        import hashlib

        def simulate_brute_force(target_hash):
            common_passwords = [
            "123456", "password", "123456789", "qwerty", "12345678", "12345", "1234567", "password123", "1234567890", "1234",
            "admin", "p@ssword", "root", "111111", "monkey", "sunshine", "iloveyou", "letmein", "secret", "football",
            "dragon", "master", "shadow", "hunter", "superman", "batman", "killer", "matrix", "hacker", "fsociety",
            "welcome", "login", "guest", "access", "system", "security", "server", "network", "firewall", "database",
            "000000", "123123", "654321", "987654321", "password1", "pass123", "abc123", "qazwsx", "wsxedc", "edcrfv",
            "chelsea", "arsenal", "liverpool", "united", "barcelona", "madrid", "soccer", "baseball", "basketball", "hockey",
            "charles", "george", "thomas", "william", "alexander", "nicholas", "matthew", "andrew", "daniel", "robert",
            "jennifer", "jessica", "michelle", "amanda", "ashley", "sarah", "stephanie", "heather", "nicole", "elizabeth",
            "princess", "angel", "sweetie", "honey", "cookie", "maggie", "charlie", "bailey", "lucky", "buster",
            "laptop", "computer", "keyboard", "monitor", "iphone", "android", "windows", "linux", "ubuntu", "kali", "elliot",
            "alderson", "elliotalderson", "verstappen", "hamilton", "starwars", "avengers", "pokemon", "superstar", "freedom", "awesome",
            "keyboard", "asdfghjk", "mnbvcxz", "1q2w3e4r", "password!", "P@ssw0rd1", "user1234", "test1234", "admin123", "manager",
            "spring2024", "summer24", "winter25", "january", "monday", "coffee", "whiskey", "tequila", "beer123", "smoke",
            "quantum", "entropy", "algorithm", "localhost", "127.0.0.1", "sysadmin", "database1", "oracle", "cisco123", "security!",
            "beowulf", "ragnarok", "anubis", "zeus123", "olympus", "viking", "sparta", "gladiator", "phoenix", "valhalla",
            "carpediem", "mementomori", "cogitoergo", "fortuna", "virtue", "justice", "liberty", "victory", "champion", "winner"
        ]

            print(f"\033[1;33m[!] INITIALIZING BRUTE FORCE ATTACK ON HASH: {target_hash}\033[0m")
            time.sleep(1)

            for password in common_passwords:
                sys.stdout.write(f"\r\033[1;34m[*] TRYING:\033[0m {password:<15}")
                sys.stdout.flush()
                
                time.sleep(random.uniform(0.3, 0.8))
                
                guess_hash = hashlib.md5(password.encode()).hexdigest()
                
                if guess_hash == target_hash:
                    print(f"\n\n\033[1;32m[+] MATCH FOUND!\033[0m")
                    print(f"\033[1;32m[+] PASSWORD:\033[0m {password}")
                    return
            
            print(f"\n\033[1;31m[-] ATTACK FAILED: Password not in dictionary.\033[0m")

        target = "6f52550186105d6a2f7c006769062947" 
        simulate_brute_force(target)

    elif choice == "firewall":
        print("""Firewall
        is Loading""")
        time.sleep(5)
        print("Firewall.framework")
        print("FAILED TO AUTO ENABLE FIREWALL!!! [Current State: Disabled][Error Code: 3435FWEE]")
        firewall = input("""Enable Or Disable Firewall Manually
        >>> """)
        print("FIREWALL STATUS: " + firewall + "[Temporary State!!!]")
      
    else:
        print("\nIllegal Command")
